models/_sector/dex/trades/base/_schema.yml

```diff
@@ -626,11 +626,30 @@ models:
       contributors: Henrystats, tomfutago
     config:
       tags: [ 'base', 'dex', 'trades', 'openocean' ]
-    description: "openocean base base trades"
+    description: "leetswap_v2 base base trades"
```

Comment:
By the way, what's the relationship between `leetswap` and `openocean`?
Leetswap's schema has already been defined in [Line 602](https://github.com/duneanalytics/spellbook/pull/5429/files#diff-60520bb14954436eba8e725c491507a5165a3f6b8a2a0fcff85af99cf6f7cb1bR602).

I think we can recover this line.

Reply:
@Hosuke it was while I was trying to resolve conflicts the lines got mixed up. Fixed the description

Reply:
resolved the conflicts as well @Hosuke